e2etest/pkg/frr/validation.go

```diff
@@ -60,9 +60,23 @@ func RoutesMatchNodes(nodes []v1.Node, route Route, ipFamily ipfamily.Family, vr
 
 		delete(nodesIPs, h.String())
 	}
+
+	excludesNodeIPs, err := k8s.NodeIPsForExcludeBalancers(nodes, ipFamily, vrfName)
```

Comment:
I'd rather have this outside (on the calling site), with an utility function that filters the nodes from the []v1.Node slice (and only in the relevant test)

Reply:
please forget it... :(